src/Client.php

```diff
@@ -91,44 +107,104 @@ protected function filterRequestBody(array $body)
 
     /**
      * @param PaymentRequest $request
-     * @return string payment request id
+     * @return array payment request id
      * @throws \GuzzleHttp\Exception\GuzzleException
-     * @throws ValidationException
+     * @throws ValidationException|CertificateException
      */
     public function createPaymentRequest(PaymentRequest $request)
     {
-        $response = $this->sendRequest('POST', '/paymentrequests', [
-            'json' => $this->filterRequestBody((array) $request),
+        $response = $this->sendRequest('POST', 'v1/paymentrequests', [
+            'json' => $this->filterRequestBody((array)$request),
         ]);
 
-        return Util::getObjectIdFromResponse($response);
+        return Util::getPaymentRequestIdsFromResponse($response);
+    }
+
+    /**
+     * Cancel a given Payment Request by its id
+     * @param string $id Payment request id to cancel
+     * @return PaymentRequest
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws ValidationException|CertificateException
+     */
+    public function cancelPaymentRequest($id)
+    {
+        $response = $this->sendRequest('PATCH', 'v1/paymentrequests/' . $id, [
+            'json' => [
+                ['op' => 'replace', 'path' => '/status', 'value' => 'cancelled']
+            ]
+        ]);
+
+        return json_decode((string)$response->getBody(), true);
     }
 
     /**
      * @param string $id Payment request id
      * @return PaymentRequest
      * @throws \GuzzleHttp\Exception\GuzzleException
-     * @throws ValidationException
+     * @throws ValidationException|CertificateException
      */
     public function getPaymentRequest($id)
     {
-        $response = $this->sendRequest('GET', '/paymentrequests/' . $id);
+        $response = $this->sendRequest('GET', 'v1/paymentrequests/' . $id);
 
         return new PaymentRequest(
-            json_decode((string) $response->getBody(), true)
+            json_decode((string)$response->getBody(), true)
         );
     }
 
+    /**
+     * @param PaymentRequest $paymentRequest
+     * @param string $instructionUUID - 32 chars uppercase in format ^[0-9A-F]{32}$. Leave blank to let PHP automatically generate a instruction UUID
+     * @return array
+     * @throws CertificateException
+     * @throws GuzzleException
+     * @throws ValidationException
+     */
+    public function createPaymentRequestV2(PaymentRequest $paymentRequest, $instructionUUID = '')
```

Comment:
This is interesting, do we need to support both the V1 and V2 version of this endpoint? I would prefer to only support the latest, if possible.